### DISCUSSION TITLE
Use IMX477 Camera with A603 Jetson Carrier Board | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1455] [Q&A]
# Use IMX477 Camera with A603 Jetson Carrier Board | Seeed Studio Wiki

use IMX477 camera with A603 Jetson carrier board

https://wiki.seeedstudio.com/Use_IMX477_Camera_with_A603_Jetson_Carrier_Board/

**Comment by Anton-Vannesjo** [comment]
Hello. Is there any solution available to use the IMX477 with the JP6.0, for the A603 carrier board?

**Comment by MassarSara** [comment]
Hello. Is there any solution available to use the IMX477 with the JP6.2, for the A603 carrier board?

**Comment by MassarSara** [comment]
I used this tutorial and it causes USB stop working.

**Comment by alirezaahmadi** [comment]
DOES NOT WORK AT ALL ,

**Reply by nfs0619** [reply]
Hi, could you please let us know which step/part doesn’t work?
Please share your device model, JetPack/L4T version, and the steps you followed along with any error logs.
You can also contact our tech support at **[techsupport@seeed.io](mailto:techsupport@seeed.io)** with the details above for faster and more specific help.

**Reply by alirezaahmadi** [reply]
Hi, thanks for reposne, 
yes we are using Jetpack 6.0 on a Orin 16gb nx - A603 carrier board. also already removed R8 from high res Pi camera. 
and the IMX477 is set up already on CSI port configuration. but still the camera does not get recognized. all devices are health checked. still we dont get video0 or anything similar on the /dev/ directory ..

**Reply by bleaaach** [reply]
Hi, could you provide the specific camera model and error logs?

**Reply by alirezaahmadi** [reply]
Here is the camera that we use:  https://www.reichelt.de/de/de/shop/produkt/raspberry_pi_-_kamera_12mp_75_c-_cs-fassung-276919?PROVID=2788&gad_source=1&gad_campaignid=1051104140&gbraid=0AAAAADwnxtYBZaR_4VfeFbPqeLR2av-HC&gclid=Cj0KCQiAvOjKBhC9ARIsAFvz5lgQ6Rqgx2YJeOgSLQnnrTQVMjYAmTLKUkqn9sgVOxd8YsAYUy90FBgaAg0NEALw_wcB

**Reply by bleaaach** [reply]
Hi,

sorry for the late reply. Could you please send an email to techsupport@seeed.io? We will get back to you as soon as possible.
